### ISSUE TITLE
Data manipulation of data inside the report

### ISSUE THREAD

**Issue by rau** [eAbsentee/eAbsentee#55]
Return map with pins in it of user sign ups - this is sooo cool, but a future issue.

**Comment by sumanthratna** [comment]
We could just write to a GeoJSON file and then in the admin panel use [https://github.com/uber/deck.gl](deck.gl) to display the map.

**Comment by rau** [comment]
![image](https://user-images.githubusercontent.com/33409793/66598899-11dbe900-eb70-11e9-838e-5168f936d535.png)

I ended up accomplishing this using the gmplot and googlemaps python packages. The gmplot package is actually pretty old, and I had to manually edit a couple parts of the library in the installed place in order to make it work, but it works now. The googlemaps package is used to convert addresses into latitudes and longitudes. It is very good at determining the correct address even when the address has a bunch of nonsense in it - including suffixes, random words, incorrect commas, etc. 

To-Do:

-  Generate Daily Files of Addresses
-  Host Somewhere?
